src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -271,14 +289,19 @@ private static boolean isJobWhitelisted(final String jobName) {
             }
             for (int i = 0; i < tagList.size(); i++) {
                 String tag = tagList.get(i).replaceAll(" ", "");
-                String[]expanded = envVars.expand(tag).split("=");
-                if (expanded.length > 1) {
+                String[] expanded = envVars.expand(tag).split("=");
```

Comment:
Should we do the same here  ? I'm confused about what this function actually does